src/SdfGenerator.cc

```diff
@@ -422,6 +426,29 @@ namespace sdf_generator
     return true;
   }
 
+  void updateModelElementWithNestedInclude(sdf::ElementPtr &_elem)
+  {
+    sdf::ElementPtr e = _elem->GetFirstElement(), nextE = nullptr;
+    while (e != nullptr)
+    {
+      nextE = e->GetNextElement();
+
+      if (e->GetIncludeElement() != nullptr)
+      {
+        std::string uri = "file://" + e->FilePath();
```

Comment:
I tested this with a Fuel model and it changed the URI to `file://`. Is there a way we can preserve the Fuel URIs or other URI schemes? I would think we can just add the include element without overriding the URI.

Reply:
I will look into this. I added `file://` to the URI since it was done here: https://github.com/ignitionrobotics/ign-gazebo/blob/797d5b90f5262d02b7041754bfd90544b32e5867/src/SdfGenerator.cc#L374-L379

Reply:
Done 1ff35d0c9032f60d5a74fb6b803944230f318ca5 and verified the generated sdf file was able to be loaded into ignition (without needing `file://`)

Reply:
In that code snippet, the model was not in `includUriMap`, so it is assumed to be a file path, so I think it's okay to prefix it with `file://`.